inspirehep/modules/migrator/tasks.py

```diff
@@ -154,6 +154,43 @@ def migrate_chunk(chunk, broken_output=None, dry_run=False):
         models_committed.connect(record_modification)
 
 
+@celery.task(ignore_result=True)
+def add_citation_counts():
+    from elasticsearch.helpers import bulk as es_bulk
+    from elasticsearch.helpers import scan as es_scan
+
+    # list of update-jsons for each record
+    records_to_update = []
+
+    # lookup dictionary where key: recid of the record and value: number of records that cite that record
+    citations_lookup={}
```

Comment:
This could be a `defaultdict`: https://docs.python.org/2/library/collections.html#collections.defaultdict